css/main.css

```diff
@@ -0,0 +1,376 @@
+}
+
+.card-link {
+  position: absolute;
+  bottom: 0;
+  left: 0;
+  right: 0;
+}
+
+li {
+  list-style: none;
+  font-family: "Inter", sans-serif;
+  font-style: normal;
+  font-weight: normal;
+  font-size: 22px;
+  line-height: 43px;
+  text-align: center;
+  color: #fff;
+  padding: 5px;
+}
+
+.card-stacks li {
+  background: rgba(255, 255, 255, 0.24) !important;
+  backdrop-filter: blur(2px);
+  -webkit-backdrop-filter: blur(2px);
+}
+
+.card h2 {
+  font-family: "Crete Round", serif;
+  font-style: normal;
+  font-weight: normal;
+  font-size: 32px;
+  line-height: 44px;
+  color: #fff;
+}
+
+.gtime {
+  background-image: url("../media/png/gtime.png");
+}
+
+.weather {
+  background-image: url("../media/png/weather.png");
+}
+
+.conference {
+  background-image: url("../media/png/conference.png");
+}
+
+.katchy {
+  background-image: url("../media/png/katchy.png");
+}
+
+.project-stacks {
+  display: flex;
+  flex-wrap: wrap;
+  grid-gap: 15px;
+  padding: 20px 0;
+}
+
+.project-stacks li {
+  display: inline;
+  border: 1px solid #8993a4;
+  padding: 10px 12px;
+  background: transparent;
+  color: #fff;
+  transition: 350ms;
+  cursor: pointer;
+}
+
+.project-stacks li:hover {
+  background: rgba(0, 0, 0, 0.349);
+}
+
+.about-me,
+.about-me-section {
+  width: 100%;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  flex-direction: column;
+}
+
+.about-me-text {
+  font-family: "Inter", sans-serif;
+  font-style: normal;
+  font-weight: normal;
+  font-size: 16px;
+  line-height: 24px;
+  color: #f4f5f7;
+}
+
+.link {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  justify-content: center;
+  padding: 12px;
+  background: #ff6b00;
+  font-family: "Inter", sans-serif;
+  font-style: normal;
+  font-weight: bold;
+  font-size: 17px;
+  line-height: 24px;
+  letter-spacing: 0.03em;
+  color: #fff;
+  align-self: start;
+  transition: 350ms;
+}
+
+.link:hover {
+  background: #a04300;
+}
+
+.about-me-icon {
+  position: absolute;
+  right: 0;
+  bottom: 0;
+}
+
+.list-title {
+  font-family: "Inter", sans-serif;
+  font-style: normal;
+  font-weight: bold;
+  font-size: 32px;
+  line-height: 44px;
+}
+
+.languages {
+  list-style: inside url("../media/svg/rectangle.svg");
+}
+
+.frameworks {
+  list-style: inside url("../media/svg/square.svg");
+}
+
+.skills {
+  list-style: inside url("../media/svg/circle.svg");
+}
+
+/* footer {
+  padding: 2rem;
+} */
+
+form {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
+
+#contact-text {
+  color: purple;
+  text-align: center;
+  margin: 1.5rem 0;
+  padding: 2rem;
+  font-family: 'Segoe UI', Tahoma, Geneva, Verdana, sans-serif;
+}
+
+.contact-input {
+  width: 90%;
+  border: none;
+  border-bottom: 1px solid grey;
+  height: 2rem;
+  margin-top: 2rem;
+  flex-direction: column;
+}
+
+#message {
+  background-color: rgb(252, 240, 240);
+  font-family: 'Segoe UI', Tahoma, Geneva, Verdana, sans-serif;
+}
```

Comment:
[OPTIONAL] Since the previous reviewer did not point this out, the following will be marked as optional in this PR.

Please, kindly see my comment above about the Figma template.

- [ ] Please, kindly see my comment above about the font.
- [ ] There should be no bottom border on the message field.
- [ ] You could set the initial height of the message field to be taller. For this, you could use the [`min-height`](https://developer.mozilla.org/en-US/docs/Web/CSS/min-height) property.

Please, kindly see the screenshots below for references.

## Figma template

![image](https://user-images.githubusercontent.com/81232315/152633787-49482974-6b88-48f2-bb3b-270372a841b7.png)

## Your version

![image](https://user-images.githubusercontent.com/81232315/152633752-b46a421d-35f4-4566-b3dd-000bf145f211.png)

---

### Additional suggestion

If you try to resize the message field, it resizes both horizontally and vertically. Ideally, the message field should resize only vertically, since we can see the resize icon on the Figma template.

Resizing horizontally is not ideal because it breaks the layout.

![image](https://user-images.githubusercontent.com/81232315/152634264-94f73963-04c7-4d09-8dec-0d57e58c0dd4.png)

- [ ] You could prevent the message field from resizing horizontally and allow only vertical resize. For this, you could use the [`resize`](https://developer.mozilla.org/en-US/docs/Web/CSS/resize) property.